src/purerpc/wrappers.py

```diff
@@ -107,12 +108,13 @@ async def __call__(self, message_aiter, *, metadata=None):
 
 
 class ClientStubStreamStream(ClientStub):
+    @asynccontextmanager
     async def call_aiter(self, message_aiter, metadata):
         stream = await self._stream_fn(metadata=metadata)
+
         async with anyio.create_task_group() as task_group:
             task_group.start_soon(send_multiple_messages_client, stream, message_aiter)
-            async for value in stream_to_async_iterator(stream):
-                yield value
+            yield stream_to_async_iterator(stream)
```

Comment:
documenting for posterity:

Previously, ClientStubStreamStream call was a generator that yielded from within a nursery / task group.  This isn't supported, see https://trio.readthedocs.io/en/v0.22.0/reference-core.html#cancel-scopes-and-nurseries

Now it's an `@asynccontextmanager` that yields at iterator.  In the case of `@asynccontextmanager`, yield is allowed from within a nursery.